src/components/sign-in/sign-in.ts

```diff
@@ -0,0 +1,65 @@
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import firebase from "firebase";
+
+@Component({
+  template: "./sign-in.html",
+  components: {}
+})
+export default class SignIn extends Vue {
+  // Data property
+  public database?: any; // Ce type e asta?
```

Comment:
```suggestion
  public database?: any; // Ce type e asta?
```
Database, usersRef, CategoriesRef - ar trebuii sa fie private ca nu le folosesti in template